cli-plugins/apps/tutorials.hbs.md

```diff
@@ -0,0 +1,126 @@
+
+    ```console
+    tanzu plugin install --local ./cli apps
+    ```
+
+2. To verify that the CLI is installed correctly, run:
+
+    ```console
+    tanzu apps version
+    ```
+
+    A version should be displayed in the output.
+
+    If the following error is displayed during installation:
+
+    ```console
+    Error: could not find plug-in "apps" in any known repositories
+
+    ✖  could not find plug-in "apps" in any known repositories
+    ```
+
+    Verify that there is an `apps` entry in the `cli/manifest.yaml` file. It should look like this:
+
+    ```yaml
+    plugins:
+    ...
+        - name: apps
+        description: Applications on Kubernetes
+        versions: []
+    ```
+
+#### <a id='from-release'></a>From Release
+
+The latest release can be found in the [repository releases page](https://github.com/vmware-tanzu/apps-cli-plugin/releases/). Each of these releases has the *Assets* section where the packages for each *system-architecture* are placed.
+
+To install the Apps CLI plug-in:
+
+Download binary executable `tanzu-apps-plugin-{OS_ARCH}-{version}.tar.gz`
+Run the following commands(for example for macOS and plugin version v0.10.0)
+
+```bash
+tar -xvf tanzu-apps-plugin-darwin-amd64-v0.10.0.tar.gz
+tanzu plugin install apps --local ./tanzu-apps-plugin-darwin-amd64-v0.10.0 --version v0.10.0
+```
+
+### <a id='uninstall'>Uninstalling Apps CLI plug-in
+
+To uninstall Apps CLI plug-in by using:
+
+```bash
+tanzu plugin delete apps
+```
+
+## <a id='changing-clusters'></a> Changing clusters
+
+The Apps CLI plug-in refers to the default kubeconfig file to access a Kubernetes cluster.
+When a `tanzu apps` command is run, the plug-in uses the default context that's defined in that kubeconfig file (located by default at `$HOME/.kube/config`).
+
+There are two ways to change the target cluster:
+
+1. Use `kubectl config use-context <context-name>` to change the default context. All subsequent `tanzu apps` commands will target the cluster defined in the new default kubeconfig context.
+2. Include the `--context <context-name>` flag when running any `tanzu apps` command. All subsequent `tanzu apps` commands without the `--context <context-name>` flag will continue to use the default context set in the kubeconfig.
+
+There are also two ways to override the default kubeconfig:
+
+3. Set the env var `KUBECONFIG=<path>` to change the kubeconfig the Apps CLI plug-in should reference. All subsequent `tanzu apps` commands will reference the non-default kubeconfig assigned to the env var.
+4. Include the  `--kubeconfig <path>` flag when running any `tanzu apps` command. All subsequent `tanzu apps` commands without the `--kubeconfig <path>` flag will continue to use the default kubeconfig.
+
+For more information about kubeconfig, see [Configure Access to Multiple Clusters](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+
+## <a id='autocompletion'></a> Autocompletion
+
```

Comment:
Use of autocomplete greatly optimizes usage of any CLI and the `apps` plugin has provided robust autocompletion support so that users do not need to struggle to remember the correct spellings and or possible entries required.

The plugin supports autocompletion for commands, positional arguments, flags and flag values and enabling this feature is highly recommended.